httpx/utils.py

```diff
@@ -95,7 +94,6 @@ def guess_json_utf(data: bytes) -> typing.Optional[str]:
 NETRC_STATIC_FILES = (Path("~/.netrc"), Path("~/_netrc"))
 
 
-@functools.lru_cache(1)
 def get_netrc(netrc_env: str) -> typing.Optional[netrc.netrc]:
```

Comment:
I'd push `netrc_env = os.getenv("NETRC", "")` inside this function, and drop the `netrc_env` argument.